.gitignore

```diff
@@ -0,0 +1 @@
+*.bacpac
```

Comment:
That reminds me:
Please create a `.dockerignore` for `docker-resources` to ignor bacpac files while building the import container
OR
move the dockerfile of mssqlimport to a different subfolder

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#exclude-with-dockerignore